test/base/EscrowClaimTokensTest.js

```diff
@@ -14,29 +14,32 @@ const DAI = artifacts.require("./mock/token/DAIMock.sol");
 const Escrow = artifacts.require("./mock/base/EscrowMock.sol");
 const LoansBase = artifacts.require("./mock/base/LoansBaseMock.sol");
 
+// Libraries
+const LoanLib = artifacts.require("../util/LoanLib.sol");
+
 contract("EscrowClaimTokensTest", function(accounts) {
   const loansEncoder = new LoansBaseInterfaceEncoder(web3);
 
   let loans;
   let instance;
 
   beforeEach(async () => {
+    const loanLib = await LoanLib.new();
+    await LoansBase.link("LoanLib", loanLib.address);
     loans = await LoansBase.new();
     instance = await Escrow.new();
   });
 
   withData({
-    _1_loan_active: [ loanStatus.Active, accounts[1], accounts[1], false, false, 0, 0, true, "LOAN_NOT_CLOSED" ],
-    _2_loan_not_liquidated_recipient_not_borrower: [ loanStatus.Closed, accounts[1], accounts[2], false, false, 0, 0, true, "LOAN_NOT_LIQUIDATED" ],
-    _3_loan_not_liquidated_recipient_is_borrower: [ loanStatus.Closed, accounts[1], accounts[1], false, false, 2, 1000, true, "LOAN_NOT_LIQUIDATED" ],
-    _4_loan_liquidated_recipient_not_borrower_caller_loans: [ loanStatus.Closed, accounts[1], accounts[2], true, true, 2, 1000, false, null ],
-    _5_loan_liquidated_recipient_is_borrower_caller_loans: [ loanStatus.Closed, accounts[1], accounts[1], true, true, 2, 1000, false, null ],
-    _6_loan_liquidated_recipient_not_borrower_caller_not_loans: [ loanStatus.Closed, accounts[1], accounts[2], false, true, 2, 1000, true, "CALLER_MUST_BE_LOANS" ],
+    _1_loan_active: [ loanStatus.Active, accounts[1], accounts[1], false, 0, 0, true, "LOAN_NOT_CLOSED" ],
+    _2_loan_not_liquidated_recipient_not_borrower: [ loanStatus.Closed, accounts[1], accounts[2], false, 0, 0, true, "LOAN_NOT_LIQUIDATED" ],
+    _3_loan_not_liquidated_recipient_is_borrower: [ loanStatus.Closed, accounts[1], accounts[1], false, 2, 1000, true, "LOAN_NOT_LIQUIDATED" ],
+    _4_loan_liquidated_recipient_is_borrower: [ loanStatus.Closed, accounts[1], accounts[1], true, 2, 1000, false, null ],
+    _5_loan_liquidated_recipient_not_borrower: [ loanStatus.Closed, accounts[1], accounts[2], true, 2, 1000, true, "CALLER_MUST_BE_LOANS" ],
```

Comment:
Why we removed a scenario?

Reply:
This function was split into 2 on the Escrow. 1 for paying out the liquidator and the other for the borrower to claim the remaining tokens.